packages/@aws-cdk/aws-dynamodb/lib/table-time-to-live-handler/utils.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+import type { OnEventRequest } from '@aws-cdk/custom-resources/lib/provider-framework/types';
+import { AWSError, DynamoDB } from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
+const dynamodb = new DynamoDB({ apiVersion: '2012-08-10' });
+
+
+export async function disableTimeToLive(event: OnEventRequest) {
+  console.log('Trying to disable time to live.');
+  try {
+    await dynamodb.updateTimeToLive({
+      TableName: event.ResourceProperties.TableName,
+      TimeToLiveSpecification: {
+        AttributeName: event.OldResourceProperties?.TimeToLiveAttribute,
+        Enabled: false,
+      },
+    }).promise();
+  } catch (err) {
+    // Catch exception so we can try disabling again at a later point in time.
+    // This is done to mimic the necessary behavior in the enable functionality.
+    const awsError = err as AWSError;
+    if (
+      awsError.code === 'ValidationException' &&
+      awsError.message === 'Time to live has been modified multiple times within a fixed interval'
```

Comment:
I wonder if this error message is guaranteed not to change...

From the docs at https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateTimeToLive.html it looks like checking only for a `ValidationException` would be enough?

@skinny85 what do you think?

Reply:
Good point. How about we just make the error message matching a little more future-proof? Something like `awsError.message.contains('Time to live') && awsError.message.contains('interval')`?

Reply:
You're right, it may have been too specific. I have updated the code to use your suggestion.